API/api/tests/test_get_available_slots.py

```
@@ -0,0 +1,160 @@
+        )
+        p2 = self.create_plage(
+            name="p2",
+            start="10:00:00",
+            end="11:00:00",
+        )
+
+        resp = self.call_api()
+        actual = self.extract(resp)
+
+        # ensure no slot violates p1 break
+        for plage_id, start, end in actual:
```

Ruff (0.15.12)

[warning] Loop control variable `end` not used within loop body

Rename unused `end` to `_end`

(B007)